dtmi/ip3/hrm-1.json

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:ip3:hrm;1",
+  "@type": "Interface",
+  "displayName": "Smart Electric Meter",
+  "description": "ip3.co.ltd",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "sn",
+      "displayName": "Device SN",
+      "description": "Device Serials Number.ex S202104150006",
+      "schema": "string"
+    },
+    {
+      "@type": "Property",
+      "name": "sw",
+      "displayName": "Software Version",
+      "description": "the device software version,ex.sw:NB.1.1.2",
```

Comment:
This and the next property I'm  wondering if the first letter should be capitalized.